ansible/group_vars/orchestration.yml

```diff
@@ -50,7 +50,11 @@ venv: "{{ orchestration_work_dir }}/venv"
 # If user specifies intalling from repository, then he can choose the specific
 # repository branch
 orchestration_repo_branch: "{{ repo_branch }}"
-orchestration_st2_installer_repo_branch: master
+
+# TODO: This is temp fix to use deprecated all in one image.
+# FIXME: Change the deprecated image to new st2-docker
+orchestration_st2_installer_repo_branch: DEPRECATED/all-in-one
```

Comment:
Is this a good idea? Esp the branch with deprecated?

Reply:
This is workaround to get the orchestration working and also the CI for installer to move. Fix to use new st2-docker is a bigger change and will take some considerable effort. I have already put a FIXME. #376 is already open for that